README.md

```diff
@@ -15,8 +15,8 @@ For more hardware-details, have a look at the [official Windows specification pa
 
 ### Install
 
-To install it you need cargo.
+Clone this repository and run `chmod +x install.sh && ./install.sh` in it, then follow the instructions.
```

Comment:
The `chmod +x` shouldn't be required if the file is marked executable in git / when committed. Same for the uninstall script.